http/headers/feature-policy.json

```diff
@@ -599,6 +599,92 @@
             }
           }
         },
+        "gamepad": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Feature-Policy/gamepad",
+            "spec_url": "https://w3c.github.io/permissions/#gamepad",
+            "support": {
+              "chrome": {
+                "version_added": "86",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "#restrict-gamepad-access",
```

Comment:
Note, this is what is listed in various docs as the permission, but I can't find it on my normal chrome. Might be some kind of Canary-only thing?

Reply:
I was able to confirm this in 86 (and not in 85) on BrowserStack. (Though for future reference, Canary-only features are `false`, at least until the `"preview"` lands.)